luarules/gadgets/unit_missile_counter.lua

```diff
@@ -0,0 +1,72 @@
+local gadget = gadget ---@type Gadget
+
+function gadget:GetInfo()
+	return {
+		name = "Missile counter",
+		desc = "Checks how many missiles are needed to kill a target (used on Legion Medusa)",
+		author = "EnderRobo",
+		date = "May 7, 2025",
+		license = "GNU GPL, v2 or later",
+		layer = 0,
+		enabled = true
+	}
+end
+
+if not gadgetHandler:IsSyncedCode() then
+	return false
+end
+
+local isCommander = {}
+
+local function missile_counter(unitID, unitDefID, team)
+	
+	local armored, armorMultiplier
+	local damage = Spring.GetUnitWeaponDamages (unitID, 2, Game.armorTypes.standard)
+	local health, maxHealth
+	local counter
+	local losstate
+	local works, isUserTarget, targetID = Spring.GetUnitWeaponTarget (unitID, 1) 
+	local burstrate = (Spring.GetUnitWeaponState (unitID, 2, "burstRate") * 1000)
```

Comment:
```suggestion
	local damage = Spring.GetUnitWeaponDamages(unitID, 2, Game.armorTypes.standard)
	local health, maxHealth
	local counter
	local losstate
	local works, isUserTarget, targetID = Spring.GetUnitWeaponTarget(unitID, 1) 
	local burstrate = (Spring.GetUnitWeaponState(unitID, 2, "burstRate") * 1000)
```
No space between function and its parameters

Reply:
Done